docs/pages/includes/commercial-prereqs-tabs.mdx

```diff
@@ -15,7 +15,7 @@
   # Teleport v(=teleport.version=) go(=teleport.golang=)
   
   $ tsh version
-  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  # Teleport Enterprise v(=teleport.version=) go(=teleport.golang=)
```

Comment:
there is not really an enterprise `tsh`, it is just the OSS `tsh`, I don't think it will print this?

Reply:
Thanks @nklaassen , corrected.

Reply:
Beat me to it... I was re-installing to confirm 😆